Source/Outposts/Outpost/Outpost.cs

```diff
@@ -285,7 +323,24 @@ public override IEnumerable<Gizmo> GetGizmos()
                 disabled = occupants.Count == 1,
                 disabledReason = "Outposts.Command.Remove.Only1".Translate()
             };
-
+            if (OutpostsMod.Settings.DeliveryMethod != DeliveryMethod.Store && !ResultOptions.NullOrEmpty())
+            {
+                yield return new Command_Action
+                {
+                    action = () =>
+                    {
+                        var menuOptions = new List<FloatMenuOption>();
+                        foreach (var map in Find.Maps.Where(m => m.IsPlayerHome).OrderByDescending(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)))
```

Comment:
```suggestion
                        foreach (var map in Find.Maps.Where(m => m.IsPlayerHome).OrderBy(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)))
```
Same as above. Doesn't matter as much here, but consistency is nice.